libs/arcade-core/arcade_core/catalog.py

```diff
@@ -973,7 +978,9 @@ def create_func_models(func: Callable) -> tuple[type[BaseModel], type[BaseModel]
         tool_field_info = extract_field_info(param)
         param_fields = {
             "default": tool_field_info.default,
-            "description": tool_field_info.description,
+            "description": tool_field_info.description
+            if tool_field_info.description
+            else "No description provided.",
```

Comment:
This is not harmful, but I would prefer to add things like default descriptions in the Engine rather than here (so we have more control over it).